src/inverseRobot.js

```diff
@@ -7,7 +7,19 @@
 */
 
 function inverseRobot(robot) {
-  // write code here
-}
+  const arrOfKeys = Object.values(robot).reverse();
+  const arrOfValue = Object.keys(robot).reverse();
+  const inverseObjectRobot = {};
+
+  const lenOfArrays = arrOfKeys.length;
```

Comment:
I am not sure, but you don't need blank lines between variables declaration

Reply:
You use the value just once, no need for extra variable.